index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>José Fernández-Caminero: Full-Stack Engineer; Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <nav>
+        <img class="artdeco1" src="resources/image_geometry_1.svg" alt="Geometrical decoration">
+        <img src="resources/ic_menu.svg" alt="Toolbar menu">
+    </nav>
+
+    <h1>José Fernández-Caminero</h1>
+
+    <p>I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
+
+    <button>Start collaboration</button>
```

Comment:
- [ ] please try to implement the button effect on hover and pressed as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/170578207-79cfffb1-60fe-4287-a492-433eec83840e.png)